packages/notifi-node-sample/package.json

```diff
@@ -14,7 +14,7 @@
     "dev": "nodemon lib/index.ts"
   },
   "dependencies": {
-    "@notifi-network/notifi-node": "^0.73.2",
+    "@notifi-network/notifi-node": "^0.73.3",
```

Comment:
Ah ... we might not need to manually modify the version. Lerna will handle version control when we publish it.

Reply:
It adopts to all version related section below